src/batch-client/index.ts

```diff
@@ -432,6 +433,28 @@ export class ZimbraBatchClient {
 			content
 		}));
 
+	public enableTwoFactorAuth = (options: EnableTwoFactorAuthInput) =>
+		this.jsonRequest({
+			name: 'EnableTwoFactorAuth',
+			body: {
+				name: {
+					_content: options.name
+				},
+				password: options.password
+					? {
+							_content: options.password
```

Comment:
not sure why we need to send everything in _content property
you can directly send it like `password: options.password`

Reply:
done

Reply:
I think passing data in `_content` is required here because of the structure like `<password>{content}</password>`